src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/StackExchangeRedis2Plus/SessionCache.cs

```diff
@@ -107,10 +108,11 @@ private void CleanUp()
                         }
                     }
                 }
+
+                Log.Finest($"SessionCache.Cleanup() removed {cleanedSessions}");
```

Comment:
Can we do an `if (cleanedSessions > 0)` here to cut down on noise? Or are you concerned that we'll miss the thread ending?

Reply:
I was kind of looking for that message to be a heartbeat for the thread, but I can go either way with it.

Reply:
Once a minute is probably fine.